description.xml

```diff
@@ -1,6 +1,6 @@
 <SchemeManager version="7">
 	<Id>pbdf</Id>
-	<Url>https://privacybydesign.foundation/schememanager/pbdf</Url>
+	<Url>https://schemes.yivi.app/pbdf</Url>
```

Comment:
I think we should wait with this specific change until https://github.com/privacybydesign/schemes-ops/issues/2 is fixed.

Reply:
Reverted this